components/ErrorPage.tsx

```diff
@@ -3,41 +3,31 @@ import Link from 'next/link';
 
 const ErrorPage = () => {
```

Comment:
This ErrorPage should serve as the return for your NotFound page inside your not-found.tsx